android/src/main/java/com/thebluealliance/androidclient/datafeed/APICache.java

```diff
@@ -47,10 +47,18 @@ public APICache(Database db) {
     public Observable<List<Team>> fetchTeamPage(int pageNum) {
         String where = String.format("%1$s >= ? AND %1$s <= ?", TeamsTable.NUMBER);
         int startNum = 500 * pageNum;
-        return Observable.just(mDb.getTeamsTable().getForQuery(
-          null,
-          where,
-          new String[]{Integer.toString(startNum), Integer.toString(startNum + 499)}));
+        return Observable.create((observer) -> {
```

Comment:
Good find on this better style. We've got to do this for all the others, but that can happen in another PR